app/app.js

```diff
@@ -142,6 +142,154 @@ app.delete("/api/gamer/:student_number", async (req, res) => {
   }
 });
 
+/**
+ * @api {get} /activity/:student_number Get Gamer Activity for specific student
+ * @apiName GetGamerActivityByStudent
+ * @apiGroup Activity
+ *
+ * @apiParam {String} student_number Student's unique number.
+ *
+ * @apiSuccess {Object} gamer_activity Gamer activity object.
+ * @apiSuccess {String} gamer_activity.student_number Student number, 8 digit integer.
+ * @apiSuccess {Number} gamer_activity.pc_number PC number.
+ * @apiSuccess {String} gamer_activity.game Game name.
+ * @apiSuccess {String} gamer_activity.started_at Datetime when the activity started.
+ * @apiSuccess {String} gamer_activity.ended_at Datetime when the activity ended.
+ *
+ * @apiError {String} 500 Server error.
+ */
+app.get("/api/activity/:student_number", async (req, res) => {
+  const { student_number } = req.params;
+  try {
+    const query =
+      "SELECT * FROM users_test.gamer_activity WHERE student_number = $1";
+    const result = await db.query(query, [student_number]);
+
+    res.json(result.rows);
+  } catch (err) {
+    console.error(err);
+    res.status(500).send(`Error finding gamer activity: ${err}`);
+  }
+});
+
+/**
+ * @api {get} /activity/all/recent Get Gamer Activity
+ * @apiName GetGamerActivity
+ * @apiGroup Activity
+ *
+ * @apiSuccess {Object} gamer_activity Gamer activity object.
+ * @apiSuccess {String} gamer_activity.student_number Student number, 8 digit integer.
+ * @apiSuccess {Number} gamer_activity.pc_number PC number.
+ * @apiSuccess {String} gamer_activity.game Game name.
+ * @apiSuccess {String} gamer_activity.started_at Datetime when the activity started.
+ * @apiSuccess {String} gamer_activity.ended_at Datetime when the activity ended.
+ *
+ * @apiError {String} 500 Server error.
+ */
+app.get("/api/activity/all/recent", async (req, res) => {
+  try {
+    const query = `SELECT * FROM users_test.gamer_activity
+      ORDER BY started_at 
+      DESC LIMIT 20;`;
```

Comment:
PostgreSQL by default sorts NULL values to be higher than any other value. This means what when we order by DESC all the null values go to the top. Could you order by `NULLS LAST` 

Something like
```    
const query = `SELECT * FROM users_test.gamer_activity
      ORDER BY started_at 
      DESC NULLS LAST
      LIMIT 20;`;
```